app/helpers/sessions_helper.rb

```diff
@@ -3,15 +3,42 @@ def log_in user
     session[:user_id] = user.id
   end
 
+  def handle_login user
+    log_in user
+    params[:session][:remember_me] == "1" ? remember(user) : forget(user)
+    redirect_to user
+  end
+
+  def remember user
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
```

Comment:
hàm này có tác dụng gì e ?

Reply:
hàm cookies.permanent.signed[:user_id] = user.id giúp lưu trữ id của user trong cookie lâu hơn phiên trình duyệt hiện tại thông qua từ khóa .permanent, .signed giúp nội dung của cookie được lưu ở dạng mã hóa một cách an toàn trước khi đặt nó trên trình duyệt ạ